Packs/CommonTypes/IndicatorTypes/reputation-Mutex.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "Mutex",
+    "version": -1,
+    "modified": "2022-08-08T08:40:28.757105856Z",
```

Comment:
shouldnt be in this PR

Reply:
Its here just until the first one is merged, after that git will ignore it.